Modules/posixmodule.c

```diff
@@ -79,6 +79,28 @@ corresponding Unix manual entries for more information on calls.");
 #  include <sys/stat.h>
 #endif /* HAVE_SYS_STAT_H */
 
+#if defined(__APPLE__)
+#include <TargetConditionals.h>
+#endif
+
+#if defined(__APPLE__) && HAVE_BUILTIN_AVAILABLE && !(TARGET_OS_OSX && __arm64__)
+#define HAVE_UTIMENSAT_RUNTIME __builtin_available(macos 10.13, ios 11, tvos 11, watchos 4, *)
+#define HAVE_FUTIMENS_RUNTIME  __builtin_available(macos 10.13, ios 11, tvos 11, watchos 4, *)
+#define HAVE_PREADV_RUNTIME    __builtin_available(macos 10.16, ios 14, tvos 14, watchos 7, *)
+#define HAVE_PWRITEV_RUNTIME    __builtin_available(macos 10.16, ios 14, tvos 14, watchos 7, *)
+#define HAVE_POSIX_SPAWN_SETSID_RUNTIME __builtin_available(macos 10.15, ios 13, tvos 13, watchos 6, *)
```

Comment:
Is this (the introduction of POSIX_SPAWN_SETSID) documented somewhere?

Reply:
I couldn't find any proper documentation, I got 10.15 by looking up which version of xnu introduced it.

https://opensource.apple.com/source/xnu/xnu-4903.270.47/bsd/sys/spawn.h.auto.html
https://opensource.apple.com/source/xnu/xnu-6153.11.26/bsd/sys/spawn.h.auto.html

Reply:
It is 10.15. I know because I asked for it (cf. FB5361314 aka rdar://43247036).